py/asmxtensa.c

```diff
@@ -150,7 +155,7 @@ void asm_xtensa_bccz_reg_label(asm_xtensa_t *as, uint cond, uint reg, uint label
     uint32_t dest = get_label_dest(as, label);
     int32_t rel = dest - as->base.code_offset - 4;
     if (as->base.pass == MP_ASM_PASS_EMIT && !SIGNED_FIT12(rel)) {
-        printf("ERROR: xtensa bccz out of range\n");
+        mp_obj_new_exception_msg_varg(&mp_type_RuntimeError, ET_OUT_OF_RANGE, MP_QSTR_bccz);
```

Comment:
These will need to actually raise the exception, ie be wrapped in `nlr_raise(...)`.

There are 5 instances here that need fixing.

Reply:
My bad, that's a copy and paste gone wrong - I copied that bit from `emit_inline_xtensa_error_exc(emit, …);` in `py/emitinlinextensa.c` - I removed the emitter bit to replace with something appropriate and forgot to raise the exception.  Thanks for letting me know!